defaults/main.yml

```diff
@@ -18,3 +17,27 @@ i2_include_plugins:
   - "nscp"
 i2_custom_constants: {}
 i2_custom_features: {}
+i2_custom_templates: {}
+i2_custom_commands: {}
+i2_remove_unmanaged_features: false
+
+i2_master: no
+i2_satellite: no
+i2_node_parent: master
+i2_api_port: 5665
+i2_ca_path: /var/lib/icinga2/ca
+i2_pki_path: /var/lib/icinga2/certs
+
+# Todo: Remove both and use ZonesDir constant
+i2_zones_dir: "{{ i2_conf_dir }}/zones.d"
+i2_global_dir: "{{ i2_zones_dir }}/global-templates"
+
+i2_ido_mysql_schema: /usr/share/icinga2-ido-mysql/schema/mysql.sql
+i2_ido_params:
+  database: icinga
+  user: icinga
+  password: icinga
+
+# Where else could this go? (debian-specific and not necessarily
+# icinga2-specific) (also not relevant for mariadb on stretch)
+mysql_defaults_file: /etc/mysql/debian.cnf
```

Comment:
Thanks. You'll find that I haven't been able to look into RH at all yet, since I'm doing all of this on a Raspberry Pi 3B+ cluster and there's no icinga2 package available in EPEL for `armhf` right now. I'll have to set-up a VM or container (or simply use the automated tests) to test that.